tic-tac-toe.js

```diff
@@ -1,7 +1,83 @@
-function TicTacToe() {
+$(document).on('ready', function() {
+  var game = new TicTacToe();
 
-}
+// click handler - keep it separate
+// we could have saved the button clicking function to a variable and then passed that variable to click
+  $(":button").click( function(){
+    if (game.turns % 2 === 0) {
+      $(this).css('background-color', 'red');
+    } else {
+      $(this).css('background-color', 'green');
+    }
+    // this gives me the class that the person has clicked on and is storing the value so I can use it elsewhere
+    var position = $(this).attr("class").split(" ")[1];
+    game.play(position, game.marker());
+  });
+});
 
-TicTacToe.prototype = {
-  
-}
+  // creates a class that initializes constructor for tictactoe class
+  // create a function for each of these functions?
+  function TicTacToe() {
+    this.board = [];
+    this.player = "player1";
+    this.turns = 0;
+  }
+
+  TicTacToe.prototype.play = function(position, marker) {
+  this.turn();
+  this.board[position] = marker;
+  this.won();
+  };
+
+  TicTacToe.prototype.turn = function(){
+    this.turns++;
+    if (this.turns % 2 === 0) {
+      this.player = "player1";
+    } else {
+      this.player = "player2";
+    }
+  };
+
+  TicTacToe.prototype.marker = function(player){
+    if (this.player === "player1") {
+      return 'x';
+    } else {
+      return 'o';
+    }
+  };
+
+  TicTacToe.prototype.getPlayerName = function(){
+    if (this.turn % 2 === 0 ) {
```

Comment:
It seems like you should be able to use the `this.player` variable to retrieve this data as opposed to using the `turn` variable to determine which player it is again.